django_rules/backends.py

```diff
@@ -69,12 +85,15 @@ def has_perm(self, user_obj, perm, obj=None):
         # Note:
         # is_active and is_superuser are checked by default in django.contrib.auth.models
         # lines from 301-306 in Django 1.2.3
-	# If this checks dissapear in mainstream, tests will fail, so we won't double check them :)
-        ctype = ContentType.objects.get_for_model(obj)
+        # If this checks dissapear in mainstream, tests will fail, so we won't double check them :)
+        # ctype = ContentType.objects.get_for_model(obj)
```

Comment:
should we just remove this line? (why comment it out here)

Reply:
Lol I left it as a reference of what was there, don't think it really matters I'd like to fully mvoe away from django-rules

Reply:
yes plsss